sql/updates/world/3.3.5/9999_99_99_99_world_335.sql

```diff
@@ -0,0 +1,14 @@
+--
+SET @GROUP_ID := 1122; -- Change with the proper ID if this is in use
```

Comment:
could you update this after https://github.com/TrinityCore/TrinityCore/blob/3.3.5/sql/updates/world/3.3.5/2019_07_24_00_world.sql got merged ?

Reply:
sure ;)

Reply:
done ;)